src/client.ts

```diff
@@ -127,6 +127,23 @@ export class FarosClient {
     }
   }
 
+  async entrySchema(graph = 'default'): Promise<any> {
```

Comment:
nit: is there any chance we can add a better return type than `Promise<any>`?

Reply:
Not really. Anyway, we'll pass it directly to the avsc lib, so the type doesn't matter.